### DISCUSSION TITLE
Cyberduck Amazon S3 Listing Directory/ Failed

### DISCUSSION THREAD

**Discussion by customkat** [iterate-ch/cyberduck#15119] [Q&A]
Hello,
    My cyberduck keeps giving me an error message that I have a listing directory/failed and says access denied. Please contact your web hosting service provider for assistance. We use this to download orders on shopify with the server Amazon S3. Is this happening with other people as well?

**Comment by tatze96** [comment]
Can you please [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), recreate the listing error, and send us the log file afterward? If you don't like to see sensitive data to be shared publicly, please send the log file with the ticket reference GH-15119 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file here, make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.

**Comment by dkocher** [comment]
You might have only permissions for a single bucket. Set the name of the bucket in the _Path_ setting of the bookmark to connect to a single bucket instead of requesting to list all buckets from S3.
